src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/WaysWeSell.tsx

```diff
@@ -68,113 +66,72 @@ const data: StepsWithImageBlackDataType[] = [
 
 export const WaysWeSell = () => {
   return (
-    <FullBleed background="black100" position="relative">
+    <FullBleed bg="black100" color="white100">
       <AppContainer>
-        <HorizontalPadding>
-          <Box mx={[-2, -4]} px={[2, 4]} py={[4, 12]}>
-            <Text
-              mb={[1, 2]}
-              variant={["lg-display", "xl", "xxl"]}
-              color="white100"
-            >
-              A sales strategy tailored to your artwork
-            </Text>
+        <Box px={[2, 4]} py={[4, 12]}>
+          <Text mb={[1, 2]} variant={["lg-display", "xl", "xxl"]}>
+            A sales strategy tailored to your artwork
+          </Text>
+
+          <Text mb={[4, 4, 6]} variant={["xs", "sm"]}>
+            A dedicated specialist works with you to select the best sales
+            option for your artwork.
+          </Text>
+
+          <DesktopLayout />
 
-            <Text mb={[4, 4, 6]} variant={["xs", "sm"]} color="white100">
-              A dedicated specialist works with you to select the best sales
-              option for your artwork.
-            </Text>
-            <DesctopLayout />
-            <MobileLayout />
-          </Box>
-        </HorizontalPadding>
+          <MobileLayout />
+        </Box>
       </AppContainer>
     </FullBleed>
   )
 }
 
-const ShelfItem: React.FC<StepsWithImageBlackDataType> = ({
-  src,
-  title,
-  text,
-  srcSet,
-}) => {
-  const { width } = useWindowSize()
-
-  const aspectRatio = BIG_IMAGE_HEIGHT / IMAGE_WIDTH
-
-  // calculating the width of the image contailer
-  // (screen width - HorizontalPadding - px of the parent - gridColumnGap) / 3
-  const collumnHeightMediumcreen =
-    ((width - 20 * 2 - 20 * 2 - 20 * 2) / 3) * aspectRatio
-  const collumnHeightBigScreen =
-    ((width - 40 * 2 - 40 * 2 - 40 * 2) / 3) * aspectRatio
```

Comment:
Interesting in that, this was a lot of work to do something that is not so hard: https://www.notion.so/artsy/Handling-Images-On-The-Web-In-Force-5c6832bf4b3e431cb4830061ce057399